datasets/imdb/README.md

```diff
@@ -3,6 +3,30 @@ pretty_name: IMDB
 languages:
 - en
 paperswithcode_id: imdb-movie-reviews
+train-eval-index:
+- config: plain_text
+  task: text-classification
+  task_id: binary_classification
+  splits:
+    train_split: train
+    eval_split: test
+  col_mapping:
+    text: text
+    label: label
```

Comment:
The column mapping goes from {source columns} -> {AutoTrain columns}, so we need to use `target` here:

```suggestion
    label: target
```